src/main/java/duke/Task.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone() { isDone = true; }
+
+    public void markAsUndone() { isDone = false; }
+
+    public static Task fromString(String data) {
+        if (data.charAt(1) == 'T') {
+            return ToDo.fromString(data);
+        }
+        switch (data.charAt(1)) {
+            case 'T':
```

Comment:
Maybe you could try to align the case and switch statements!

Reply:
I agree, I think the style guide wants us to do that as well.